packages/form-validations/README.md

```diff
@@ -0,0 +1,100 @@
+# @lmc-eu/spirit-form-validations
+
+> Form validations for Web and Web-Twig packages of the Spirit Design System.
+
+## Install
+
+```shell
+yarn add @lmc-eu/spirit-form-validations
+```
+
+or
+
+```shell
+npm install --save @lmc-eu/spirit-form-validations
+```
+
+## Usage
+
+Include the JavaScript file in your HTML `<head>` or just before the closing `<body>` tag.
+
+```html
+<script src="dist/bundles/umd/spirit-form-validations.min.js" type="text/javascript"></script>
+```
+
+Or
+
+```html
+<script src="dist/bundles/esm/spirit-form-validations.js" type="module"></script>
+```
+
+Now create some form and validate
```

Comment:
Can we have an example of such form please? Nothing difficult, just plain HTML like:

```html
<form id="form1">
  <label for="email">Email</label>
  <input type="email" id="email" required />
  <button>Subscribe</button>
</form>
```